### ISSUE TITLE
Picture selection issue

### ISSUE THREAD

**Issue by souragar** [zhihu/Matisse#590]
It is quite difficult to click on the small circle to select a picture. Is there any way I can disable preview and let user select pictures on click on the thumbnail?

**Comment by Isaackingston** [comment]
Yes the usability will be great if we can have a flag to enable/disable the preview so users can tap and select the images (grid item) needed. Its really painful to tap on the small check box to choose an image. @REBOOTERS  Do you think that a PR for this feature can be approved? Please tag active moderators if you think they can talk a call. @zhihu Please advise.

**Comment by neobie** [comment]
I need this too.

**Comment by matthewkrueger** [comment]
I'm not going to be able to use this library unless the preview can be disabled. I can't imagine many/any use cases where users would want to preview each individual image and would actually realize they need to click that tiny circle in the top right corner to select the image.

**Comment by matthewkrueger** [comment]
Looks like 0.5.3-beta3 adds a "showPreview(false)" configuration option. Setting that to false eliminates the preview, which makes this library (finally) usable - yay! Thanks devs, we'll be using this in our prod apps soon.
